_episodes/05-cond.md

```diff
@@ -147,7 +147,9 @@ seemed to rise like a straight line, one unit per day.
 We can check for this inside the `for` loop we wrote with the following conditional:
```

Comment:
As @annefou says, it could be confusing to refer to the `for` loop from the previous episode. Also, if the changes presented in this PR are merged, the conditional would be preceded by two variable definitions that could be confusing when the narrative says "the following conditional."

Also, I remember in our last workshop, one of the learners asked for clarification on what checking `if numpy.max(data, axis=0)[0] == 0 and numpy.max(data, axis=0)[20] == 20` was doing. We could remove what is in line 147 and make the narrative more descriptive of what we are checking with text more along the lines of:

```
Since it's unlikely that for each day of the study the maximum level of inflammation would change by exactly one unit, let's check for this condition using a conditional statement. First we will get the maximum inflammation values for the first day and after twenty days. If in both cases the value is equal to the number of days that have passed since the study began, we will print an alert to indicate we should later look more closely at our data for errors.
```

Reply:
You are right @maxim-belkin , we are only checking that the maximum value of inflammation is equal to the number of days that have passed on the study on two specific days not actually that the max changes one unit on all days. I should have used words more like "let's do a quick check for the possibility of this condition." I certainly don't expect my exact wording to be used; I more meant to indicate that I have observed the lack of explanation of what the conditional is testing to also cause confusion in the past and if the narrative is being changed, perhaps more detail could be included. :)